packages/amplify-graphql-index-transformer/src/resolvers/generators/rds-vtl-generator.ts

```diff
@@ -0,0 +1,80 @@
+import { TransformerContextProvider, TransformerResolverProvider } from '@aws-amplify/graphql-transformer-interfaces';
+import {
+  Expression,
+  printBlock,
+  compoundExpression,
+  set,
+  ref,
+  list,
+  qref,
+  methodCall,
+  str
+} from 'graphql-mapping-template';
+import { PrimaryKeyDirectiveConfiguration } from '../../types';
+import _ from 'lodash';
+import {
+  addIndexToResolverSlot,
+  getResolverObject,
+  mergeInputsAndDefaultsSnippet,
+  validateSortDirectionInput
+} from '../resolvers';
+import {
+  PrimaryKeyVTLGenerator,
+} from "./vtl-generator";
+
+export class RDSPrimaryKeyVTLGenerator implements PrimaryKeyVTLGenerator {
```

Comment:
Can we name the classes as '*IndexVTLGenerator' to be in sync with package name? We would use these classes for @index as well.